cmd/kubelet/app/options/options.go

```diff
@@ -90,6 +90,16 @@ type KubeletFlags struct {
 	// additional configs to overwrite what is provided by default and in the KubeletConfigFile flag
 	KubeletDropinConfigDirectory string
 
+	// maxAllowableNUMANodes specifies the maximum number of NUMA Nodes that
+	// the TopologyManager supports on the underlying machine.
+	//
+	// At present, having more than this number of NUMA Nodes will result in a
+	// state explosion when trying to enumerate possible NUMAAffinity masks and
+	// generate hints for them. As such, if more NUMA Nodes than this are
+	// present on a machine and the TopologyManager is enabled, an error will
+	// be returned and the TopologyManager will not be loaded.
+	MaxAllowableNUMANodes int
```

Comment:
For top-level internal flags this name is mostly OK - it should however perhaps have the `TopologyManager` prefix, let's review how other related flags are named. OTOH if you move towards topology manager policy (which I'd be generally in favor of) you can keep this name internally.

Reply:
yes, If we consider it as a policy option, It should have the `TopologyManager` prefix.

Reply:
As a policy option it should *not* have the `TopologyManager` prefix.

Reply:
Sorry for my bad words, yes, we should keep `MaxAllowableNUMANodes`  name internally.